library/Backends/llvm/stack.yaml

```diff
@@ -9,6 +9,7 @@ packages:
 - ../../Frontend
 - ../../Sexp
 - ../../Translate
+- ../../Test/DataStructures
```

Comment:
This is unrelated to the rest of the PR, right?

Reply:
Yes, but needed to actually run `stack test` on the LLVM side